CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
+There are three main ways to contribute to the project.
+Read the respective section to find out more.
 
-### Good to know
-* **Be patient**. The dev team is small and resource limited. Devs finding their free time, analyzing the problem and fixing the issue, it all takes time. :clock3:
-* If you can code, why not become a **contributor** by fixing the issue and open a pull request? :wink:
-* Harsh words or threats won't help your situation. What's worse, your complain will (very likely) to be **ignored**. :fearful:
+#### Table Of Contents
+
+[Submitting an issue/bug report](#submitting-an-issuebug-report)
+  * [Bug reporting etiquette](#bug-reporting-etiquette)
+  * [What is and actual bug report?](#what-is-and-actual-bug-report)
+  * [Before submitting a bug report](#before-submitting-a-bug-report)
+  * [Steps to ensure a good bug report](#steps-to-ensure-a-good-bug-report)
+
+[Suggesting Enhancements/feature requests](#suggesting-enhancementsfeature-requests)
+  * [Before submitting an enhancement/feature request](#before-submitting-an-enhancementfeature-request)
+  * [Steps to ensure a good enhancement/feature suggestion](#steps-to-ensure-a-good-enhancementfeature-suggestion)
+
+[Opening a pull request](#opening-a-pull-request)
+  * [Must read](#must-read)
+  * [Good to know](#good-to-know)
+
+
+# Submitting an issue/bug report
+
+This section guides you through submitting an issue/bug report for qBittorrent.
+
+Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Make sure to follow these rules carefully when submitting a bug report. Failure to do so will result in the issue being closed.
+
+## Bug reporting etiquette
+
+*   Issues and comments on issues must always be in **English**.
+
+*   This project is supported by volunteers, do not expect "customer support"-style interaction.
+
+*   **Be patient**. The development team is small and resource limited. Developers and contributors take from their free time to analyze the problem and fix the issue. :clock3:
+
+*   Harsh words or threats won't help your situation. What's worse, your complain will (very likely) be **ignored**. :fearful:
+
+## What is and actual bug report?
+
+Developers and contributors are not supposed to deal with issues for which little to no investigation to find the actual cause of a purported issue was made by the reporter.
+
+Positive contributions are those which are reported with efforts to find the actual cause of an issue, or at the very least efforts were made to narrow it as much as possible.
+
+Requiring people to investigate as much as possible before opening an issue will more than likely avoid burdening the project with invalid issues or issues unrelated to qBittorrent.
+
+The following are _not_ bug reports; **Check the wiki, forum or other places for help and support for issues like these**:
+
+-   Explanation of qBittorrent options (see wiki);
```

Comment:
it's unusual for me to see semicolons here and below

Reply:
Ok, sentence is now:
`The following are _not_ bug reports. **Check the wiki, forum or other places for help and support for issues like these**:`

Also remove trailing `;` in all bullet points